lms/static/js/instructor_dashboard/membership.js

```diff
@@ -42,7 +42,7 @@ such that the value can be defined later than this assignment (file load order).
                 add_btn_label: 'Add Member',
                 add_handler: function() {}
             });
-            templateHtml = window.Mustache.render($('#member-list-widget-template').html(), memberListParams);
+            templateHtml = _.template($('#membership-list-widget-tpl').html())(memberListParams);
```

Comment:
It would be nice to use HtmlUtils.template here (I imagine otherwise we get a safe template warning). But for the first attempt at conversion, I followed existing patterns in instructor_dashboard.